app/controllers/ontologies_controller.rb

```diff
@@ -52,6 +52,7 @@ def create
 
   def show
     @content_object = :ontology
+    params[:id] = params[:ontology_id] if params[:ontology_id]
```

Comment:
which URL did you call that only sets `params[:ontology_id]` and not `params[:id]`